scripts/run-jest/index.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn;
```

Comment:
What is this file (add comments)? And if it's common it should live somewhere common so we don't have to duplicate it across repos.

Reply:
Its a nodejs module, https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options